modular_skyrat/modules/customization/modules/surgery/organs/genitals.dm

```diff
@@ -65,17 +65,17 @@
 
 /obj/item/organ/genital/penis/get_description_string(datum/sprite_accessory/genital/gas)
 	var/returned_string = ""
-	var/pname = lowertext(genital_name)
+	var/pname = lowertext(genital_name) == "nondescript" ? "" : lowertext(genital_name) + " "
 	if(sheath != SHEATH_NONE && aroused != AROUSAL_FULL) //Hidden in sheath
 		switch(sheath)
 			if(SHEATH_NORMAL)
 				returned_string = "You see a sheath."
 			if(SHEATH_SLIT)
 				returned_string = "You see genital slit."
 		if(aroused == AROUSAL_PARTIAL)
-			returned_string += " There's a [pname] penis poking out of it."
+			returned_string += " There's a [pname]penis poking out of it."
```

Comment:
won't this urh, maked it "knottedweenie" instead of "knotted weenie"?

Reply:
Nope, because of the ternary if operator when it defines pname.